config/application.rb

```diff
@@ -1,6 +1,21 @@
 # frozen_string_literal: true
 require_relative 'boot'
-require 'rails/all'
+# We don't use:
+# require 'rails/all'
+# Instead we only load the components we need. When updating rails versions you can
+# checkout the contents of https://github.com/rails/rails/blob/main/railties/lib/rails/all.rb
+# to find out what's included by default.
+require 'active_record/railtie'
+require 'active_storage/engine'
+require 'action_controller/railtie'
+require 'action_view/railtie'
+require 'action_mailer/railtie'
+require 'active_job/railtie'
+require 'action_cable/engine'
+require 'action_mailbox/engine'
+require 'action_text/engine'
+require 'rails/test_unit/railtie'
+# require 'sprockets/railtie'
```

Comment:
I guess this comment can be removed?

Reply:
I left it in there for reference ; it is part of the standard 'rails/all' inclusions and I wanted the two to be comparable to assist with upgrades. Sprockets is dropped in the default Rails 7 install though, so will definitely be able to remove it then.

We can probably also kill a few others of these, but that is probably for another PR.